docs/gkehub_v1/configmanagement_v1/types.rst

```diff
@@ -1,7 +1,7 @@
 Types for Google Cloud Gkehub Configmanagement v1 API
 =====================================================
 
-.. automodule:: google.cloud.gkehub.configmanagement_v1.types
+.. automodule:: google.cloud.gkehub_v1.configmanagement_v1.types
```

Comment:
I'm fine with this part of the change.  Is the rename / move of the `*.rst` files just an artifact of tweaking the codegen?

Reply:
I had a bug in owlbot.py where this replacement was missing. Here is the code that touches this specific line. https://github.com/googleapis/python-gke-hub/pull/64/files#diff-5ffcc5c595bdb5127f82e03dbcdb1eeb75a2633f6f44b7f640f9d7523bc0f264R41